Class-3/Class Work/practise_set_1.js

```diff
@@ -0,0 +1,87 @@
+let mainElement = document.getElementsByClassName("mainComponent"); // it returns a html collection.
+console.log(mainElement);
+
+console.log(typeof (document.body)) //object
+console.dir(document.body)
+
+document.title = "Manipulating the Title from JS"
+for (let element of document.all) {
+    console.log(element)
+}
+
+//all elements are object
+console.log(document.getElementById("tableContent"));
+
+//changing the content of header element
+let headerElement = document.getElementById("header");
+headerElement.textContent = "Changed from JS using TC";
+headerElement.innerText = "Changed from JS using IT"
+headerElement.innerHTML = "Changed from JS using IH"
+
+console.log(headerElement.textContent); // it shows, what text is written in the html code
+console.log(headerElement.innerHTML);// it gives the entire html code
+console.log(headerElement.innerText);// it shows, what is shown in browser after rendering
+
+headerElement.style.fontSize = '15px'
+
+let para = document.getElementById("paraID")
+let items = para.getElementsByClassName("button") // only changes the classes of paraID. Interesting
+for (let i = 0; i < items.length; i++) {
+    items[i].style.color = "blue"
+}
+
+let tags = document.getElementsByTagName("td");
+for (let i = 0; tags.length; i++) {
```

Comment:
I think this loop will lead us to an Infinity ;) Here we missed `i < tags.length`, right?